api-report/purchases-js.api.md

```diff
@@ -176,6 +194,7 @@ export interface Price {
 // @public
 export interface PricingPhase {
     readonly cycleCount: number;
+    readonly period: Period | null;

```

Comment:
This is the convenience accessor added in this PR. We still keep the `periodDuration` as a ISO8601 string.